src/main/java/org/folio/inventory/dataimport/consumers/DataImportKafkaHandler.java

```diff
@@ -122,8 +122,10 @@ public Future<String> handle(KafkaConsumerRecord<String, String> record) {
           .orElse(CompletableFuture.failedFuture(new EventProcessingException(format("Job profile snapshot with id '%s' does not exist", jobProfileSnapshotId)))))
         .whenComplete((processedPayload, throwable) -> {
           if (throwable != null) {
+            LOGGER.error(throwable.getMessage());
             promise.fail(throwable);
           } else if (DI_ERROR.value().equals(processedPayload.getEventType())) {
+            LOGGER.error("Failed to process data import event payload: {}", processedPayload.getEventType());
```

Comment:
we considered using 'WARN' level for exceptions that we handle

Reply:
done